### ISSUE TITLE
Type Mismatch Error

### ISSUE THREAD

**Issue by yunboouyang** [ninia/jep#261]
**Describe the bug**
I try to use jep in my application to incorporate sklearn in my scala code base. I try to use NDArray to pass a scala array to python but get the following error. I haven't found the root cause. Could you provide any insights?
jep.JepException: <class 'ValueError'>: Expected array-like (array or non-string sequence), got <jep.PyJObject object at 0x7f20c1f16060>

**To Reproduce**

This is my scala code snippet:

    val scores = new NDArray(Array(1.0, 2.0, 3.0, 4.0), 4)
    val labels = new NDArray(Array(0.0, 1.0, 1.0, 1.0), 4)
    val interp = new SharedInterpreter()
    var auc = 0D
    try {
        interp.exec("from sklearn.metrics import roc_auc_score")
        interp.set("scores", scores)
        interp.set("labels", labels)
        interp.exec("auc = roc_auc_score(y_score=scores, y_true=labels)")
        auc = interp.getValue("auc").toString.toDouble
     } finally {
        interp.close()
     }

**Environment (please complete the following information):**
- OS Platform, Distribution, and Version: Red Hat Linux 7
- Python Distribution and Version: 3.6
- Scala Distribution and Version: 2.11
- Java Distribution and Version: 1.8
- Jep Version: 3.9.0
- Python packages used: sklearn

**Comment by bsteffensmeier** [comment]
To use NDArray from Jep you must have numpy installed before jep is installed so that jep will be built with numpy support. If you have numpy now then reinstalling jep should resolve your issue. When installing jep the output should include `numpy include found at...` and not `numpy not found, running without numpy support`

**Comment by yunboouyang** [comment]
Thank you! I've already installed numpy. Both numpy and jep are in virtual environment site-packages. When installing jep, there is no output. Jep also runs fine if the python code has numpy dependency. The error occurs only when I want to use `NDArray`. Could you shed light on the possible cause?

**Comment by bsteffensmeier** [comment]
Please verify that Jep is built with numpy support, you can do that with some code like this, which should get you a value of 1:

```
interp.exec("import jep");
inerp.getValue("jep.JEP_NUMPY_ENABLED")
```

You an also check the build logs I mentioned above by adding -v when installing. A command like this should show output indicating if numpy is installed at build time or not:

```
pip install -v --force-reinstall --ignore-installed --no-binary :all: jep
```

**Comment by yunboouyang** [comment]
Thanks! I've checked Jep is not built with numpy. I'm fixing it now.
